ui/admin/app/templates/scopes/scope/host-catalogs/host-catalog/hosts/index.hbs

```diff
@@ -22,52 +22,49 @@
 
   <page.body>
     {{#if @model.hosts}}
-      <Rose::Table as |table|>
-        <table.header as |header|>
-          <header.row as |row|>
-            <row.headerCell>{{t 'form.name.label'}}</row.headerCell>
-            <row.headerCell>{{t 'form.type.label'}}</row.headerCell>
-            <row.headerCell>{{t 'form.id.label'}}</row.headerCell>
-            <row.headerCell>{{t 'form.address.label'}}</row.headerCell>
-          </header.row>
-        </table.header>
 
-        <table.body as |body|>
-          {{#each @model.hosts as |host|}}
-            <body.row as |row|>
-              <row.headerCell>
-                {{#if (can 'read model' host)}}
-                  <LinkTo
-                    @route='scopes.scope.host-catalogs.host-catalog.hosts.host'
-                    @model={{host.id}}
-                  >
-                    {{host.displayName}}
-                  </LinkTo>
-                {{else}}
-                  {{host.displayName}}
-                {{/if}}
-                <p>{{host.description}}</p>
-              </row.headerCell>
-              <row.cell>
-                <HostCatalogTypeBadge @model={{host}} />
-              </row.cell>
-              <row.cell>
-                <Copyable
-                  @text={{host.id}}
-                  @buttonText={{t 'actions.copy-to-clipboard'}}
-                  @acknowledgeText={{t 'states.copied'}}
+      <Hds::Table
+        @model={{@model.hosts}}
+        @columns={{array
+          (hash label=(t 'form.name.label'))
+          (hash label=(t 'form.type.label'))
+          (hash label=(t 'form.id.label'))
+          (hash label=(t 'form.address.label'))
+        }}
+      >
+        <:body as |B|>
+          <B.Tr>
+            <B.Td>
+              {{#if (can 'read model' B.data)}}
+                <LinkTo
+                  @route='scopes.scope.host-catalogs.host-catalog.hosts.host'
+                  @model={{B.data.id}}
                 >
-                  <code>{{host.id}}</code>
-                </Copyable>
-              </row.cell>
-
-              <row.cell>
-                <code>{{host.address}}</code>
-              </row.cell>
-            </body.row>
-          {{/each}}
-        </table.body>
-      </Rose::Table>
+                  {{B.data.displayName}}
+                </LinkTo>
+              {{else}}
+                {{B.data.displayName}}
+              {{/if}}
+              <p>{{B.data.description}}</p>
```

Comment:
Can we use [HDS::Text component](https://helios.hashicorp.design/components/text) here?

Reply:
@calcaide yes good call out

Reply:
I am not understanding what do you mean by `The default for this Hds::Text::Body that I used here is a span tag` but I see the current `@tag` is using a `p`, that makes sense to me, because the HTML element that was used here before was a `p`, so shouldn't be any difference?

Reply:
@calcaide apologize for closing this out before I replied to your questions.  I was just making reference to using the `@tag`  to keep some of our styling that happens with our `p` element.